hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientPartitionServiceImpl.java

```diff
@@ -226,14 +244,8 @@ public void run() {
                 return;
             }
 
-            Connection connection = getOwnerConnection();
```

Comment:
Removing this check may cause periodic printing of log at line 240. Assume no cluster connection, and consider what happens.